app/serializers/discussion_reader_serializer.rb

```diff
@@ -9,5 +10,13 @@ class DiscussionReaderSerializer < ActiveModel::Serializer
              :read_items_count,
              :read_salient_items_count,
              :last_read_sequence_id,
-             :volume
+             :volume,
+             :is_read
+  def is_read
```

Comment:
Or with the query we're using to fetch the thing in the first place? VisibleDiscussions will give you read_counts etc